tests/integration/security/testdata/authz/v1beta1-negative-match.yaml.tmpl

```diff
@@ -34,6 +49,22 @@ spec:
     - source:
         notNamespaces: ["{{ .Namespace }}"]
 ---
+# The following policy won't allow request to host "deny.com"
```

Comment:
`# The following policy allows request if the host is not "deny.com"` and add a newline below (just to be consistent with other examples in this file)